devicetypes/qubino/qubino-din-dimmer.src/qubino-din-dimmer.groovy

```diff
@@ -139,6 +139,7 @@ def installed() {
 			state.currentPreferencesState."$it.key".status = "synced"
 		}
 	}
+	readConfigurationFromTheDevice()
```

Comment:
Can you explain why you've chosen to query the device like this on install?

Are configuration settings durable over an install/remove/install? I always thought that removing a device from a network would return it to defaults, but I may be wrong.

Reply:
Qubino dimmers (DIN, Flush, Flush 0-10V) don't return to their defaults after removing them from a network. Additionally - manufacturer asks (in user manuals) to remove and add the device again if setting: ~"input type" is changed.
That's why it may be needed to read the settings from the device.